test/integration/acme/orders_controller_test.go

```diff
@@ -249,7 +248,9 @@ func TestAcmeOrdersController(t *testing.T) {
 	// Reason field on Order's status. Change this test once we are setting
 	// Reasons on intermittent Order states.
 	var pendingOrder *cmacme.Order
-	err = wait.Poll(time.Millisecond*200, acmeorders.RequeuePeriod, func() (bool, error) {
+	timeoutCtx, timeoutCancel := context.WithTimeout(ctx, acmeorders.RequeuePeriod)
```

Comment:
Here a child-context is created with a specific timeout. This means that the test will fail if the ctx reaches its deadline OR if timeoutCtx reaches its deadline.

Reply:
Here you use `RequeuePeriod` but below you re-use this context to implement the timeout for a Poll which originally had a timeout set to `RequeuePeriod*3`. If requeueperiod is used here to allow time for at least one requeue to happen, then shouldn't the timeout be set to higher?